addons/rose/addon/components/rose/dropdown/index.hbs

```diff
@@ -20,7 +20,7 @@
     <span class='rose-dropdown-trigger-text'>
       {{@text}}
       {{#if @count}}
-        <Rose::Badge>{{@count}}</Rose::Badge>
+        <Hds::BadgeCount @text={{@count}} @size='small' />
```

Comment:
HDS has a [badgeCount](https://design-system-components-hashicorp.vercel.app/components/badge-count) component that I thought fit here. Added an image of this in the description.